checkstyle/checkstyle-suppressions.xml

```diff
@@ -25,7 +25,10 @@
   <suppress checks="ConstantName"
             files="ApplicationStatus.java"
             lines="26-29"/>
+  <!-- don't require FQDNs to be used in API javadocs -->
   <suppress id="preventJavadocUnusedImports"
             files=".*samza-api.*"/>
+  <!-- don't require FQDNs to be used in descriptor javadocs -->
+  <suppress files="samza-kafka/src/main/java/org/apache/samza/system/kafka/descriptors" id="preventJavadocUnusedImports" />
```

Comment:
why do we allow unused imports?

Reply:
In the descriptor classes, the javadocs may include references to some other classes which aren't directly used in the code (e.g. through `{@link <className>}`). For API classes like descriptors, it is nice to be able to just use the simple class name (e.g. `{@link MyClass}`) instead of the full qualified name (e.g. `{@link org.apache.samza.MyClass}`).
I will clarify the the comment.